gradle.properties

```diff
@@ -0,0 +1,12 @@
+systemProp.sonar.host.url=https://sonarcloud.io
+
+systemProp.sonar.projectKey=Alvearie_hri-flink-pipeline-core
+systemProp.sonar.organization=alvearie
+
+# Path is relative to the gradle.properties file. Replace "\" by "/" on Windows.
+
+sonar.coverage.jacoco.xmlReportPaths=./testCoverage.out
+#systemProp.sonar.go.tests.reportPaths=
```

Comment:
Verify in the Sonar cloud analysis for this PR, that this is working correctly. You should be able to see code coverage stats and ensure it's not including test code.